src/spyglass/position/v1/position_dlc_project.py

```diff
@@ -340,6 +340,9 @@ def insert_new_project(
         }
         # TODO: make permissions setting more flexible.
         if set_permissions:
+            raise NotImplementedError(
```

Comment:
@samuelbray32 ?

Reply:
I just removed this since it didn't seemed to be used. I suspect it was put in place by @dpeg22 to be developed in the future. We can add back if it becomes relevant again.